recipe/meta.yaml

```diff
@@ -24,38 +28,35 @@ build:
     - MIGRATING={{ migrating }}
 
 requirements:
-  {% if not win %}
-  build:
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - ipython >=7.23.1                       # [build_platform != target_platform]
-    - jupyter_client >=6.1.12                # [build_platform != target_platform]
-    - packaging                              # [build_platform != target_platform]
-    - psutil                                 # [build_platform != target_platform]
-    - python                                 # [build_platform != target_platform]
-  {% endif %}
   host:
     - debugpy >=1.0.0
     - ipython >=7.23.1
     - jupyter_client >=6.1.12
     - packaging
     - pip
     - psutil
-    - python
+    - python 3.9
   run:
-    - appnope  # [osx]
+    {% if noarch_platform == "unix" %}
+    - __unix
+    - appnope
+    {% else %}
+    - __win
+    {% endif %}
     - debugpy >=1.0
     - ipython >=7.23.1
     - jupyter_client >=6.1.12
     - matplotlib-inline >=0.1
     - nest-asyncio
     - packaging
     - psutil
-    - python
+    - python >=3.7
     - tornado >=6.1
     - traitlets >=5.1.0
 
 test:
   requires:
+    - python 3.8
```

Comment:
Why is 3.8 needed specifically for testing?

Reply:
i think the intent is demonstrate "the kernelspec works when used on another python"... but this could be `{{ min_python }}` as well...

Reply:
> i think the intent is demonstrate "the kernelspec works when used on another python"

Yup. Open to other approaches - whatever guarantees `requirements.host.python != test.requires.python`.

>  but this could be {{ min_python }} as well...

As long as we also ensure the host dependency does _not_ use `{{ min_python }}`, that should be okay. I'm not sure there's a downside to specific versions, though.